lib/constable/markdown.ex

```diff
@@ -1,9 +1,80 @@
 defmodule Constable.Markdown do
   alias Earmark.Options
 
+  defmodule CustomScrubber do
```

Comment:
Not sure if it's worth doing this or not, but what do you think about adding more tests to the `CustomScrubber` as to what it scrubs and what it doesn't? 

I know we have some tests, and I see the comment says that this is basically a copy of the `markdown_html.ex` scrubber, but might it be worth checking since we own the code now?

Reply:
That's a good idea ... I will make a second pass at this where I at least *attempt* to borrow less from the other lib and just put our own rules (along with our own tests) in place, and see how it works out. If that turns out to be too clunky or feels like I'm just duplicating the test suite of the html_sanitizer_ex lib's various scrubbers, I'll back off and probably just keep something like this.

I also thought about putting the `CustomScrubber` in it's own file with an even bigger comment at the top about where it came from and what very specific changes I made, just to be totally clear.